content/en/xlt/release-notes/8_6_x.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 8.6.x
+linkTitle: 8.6.x
+
+weight: 62
+type: docs
+
+date: 2025-01-13
+
+description: >
+    Custom Data Logger
+
+sitemap:
+  changefreq: never
+  priority: 0.1
+---
+
+## XLT 8.6.0
+
+### Framework
+
+#### Custom Data Logger
+
+When load testing, we often need to pass test data such as usernames, SKUs, coupon codes, or the like. Sometimes we want to know in retrospect which data items were actually used during a load test. Typically, this is solved by having the test scenario write the values to the logs or to a custom file. If we wanted to analyze the data used, we would first have to download the test result dataset and somehow extract the data from the log or data files.
+
+In this version of XLT, we have added the new *Custom Data Logger* feature, which can replace any custom solution in most cases. This feature allows you to log custom data lines in different scopes using a simple API. A scope is a custom string that describes the data to log, for example "users" or "skus". All data lines logged in a given scope are grouped together.
+
+A data line must be a string, but the format of the line is unspecified. It can be a single value, a structured data record such as CSV, or any other custom format. You can specify the extension of the resulting data files to reflect the format used (`*.log` by default). You can also specify a file header. See below for an example of how to set up the data logger for the "users" scope to use a CSV file:
+
+```java
+// one-time setup before first use
+DataLogger logger = Session.getDataManager().dataLogger("users");
```

Comment:
`Session.getDataManager().dataLogger("users");`
(not static!)

Reply:
Nice catch!